app/src/main/java/vn/edu/usth/minigh/PRActivity.java

```diff
@@ -11,9 +11,9 @@
 import androidx.fragment.app.FragmentManager;
 import androidx.fragment.app.FragmentTransaction;
 
-public class prActivity extends BaseActivity {
+public class PRActivity extends BaseActivity {
```

Comment:
You need to do this in the manifest XML as well.

Reply:
Thanks, fixed that.